README

```diff
@@ -95,10 +99,23 @@ REQUIREMENTS:
         Required for CPUQuota= in resource control unit settings
           CONFIG_CFS_BANDWIDTH
 
-        Required for IPAddressDeny= and IPAddressAllow= in resource control
+        Required for IPAddressDeny=, IPAddressAllow=, IPIngressFilterPath=,
+        IPEgressFilterPath= in resource control unit settings
         unit settings
+          CONFIG_BPF
+          CONFIG_BPF_SYSCALL
+          CONFIG_BPF_JIT
+          CONFIG_HAVE_EBPF_JIT
           CONFIG_CGROUP_BPF
 
+        Required for AllowBindPorts= in resource control unit settings
+          CONFIG_BPF
+          CONFIG_BPF_SYSCALL
+          CONFIG_BPF_JIT
+          CONFIG_HAVE_EBPF_JIT
+          CONFIG_CGROUP_BPF
+          CONFIG_DEBUG_INFO_BTF
```

Comment:
Is it really needed for `AllowBindPorts`? 

The eBPF programs on this PR use `struct bpf_sock_addr`, it's part of the API and shouldn't change from version to version, hence BTF is not needed, right?